src/main/resources/db/changelog/changes/07-create-categories-table.yml

```diff
@@ -12,18 +12,16 @@ databaseChangeLog:
                   autoIncrement: true
                   constraints:
                     primaryKey: true
-                    nullable: false
               - column:
                   name: name
                   type: varchar(255)
                   constraints:
                     nullable: false
+                    unique: true
               - column:
                   name: description
                   type: varchar(500)
               - column:
                   name: is_deleted
                   type: boolean
                   defaultValueBoolean: false
-                  constraints:
-                    nullable: false
```

Comment:
let's not change already applied scripts, if you need to modify the table - add another `alter table` script

Reply:
Okey, my bad. I will use alter tables in next time